tests/unit/adapters/application-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('adapter:application', 'Unit | Adapter | application', {
+  // Specify the other units that are required for this test.
+  // needs: ['serializer:foo']
+});
+
```

Comment:
It's possible I'm missing something but it seems like these tests might have been just generated as placeholders? The routes and everything look great - really clean and easy to read. But if it's possible to do something like the feature test + VCR combo from rails with ember this would be a good spot for it.